src/main/java/nl/knaw/dans/bridge/plugin/lib/common/IResponseData.java

```diff
@@ -0,0 +1,28 @@
+package nl.knaw.dans.bridge.plugin.lib.common;
+
+import java.util.Optional;
+
+/*
+ * Created by Eko Indarto
+ */
```

Comment:
All methods return optional strings, but I guess that these must be meaningful and are possibly restricted to a certain format or given set?

Reply:
👍